scripts/run_check_release_notes

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+.+\*\(.+\)\*"
+if [ $? != "0" ]
+then
+	L=git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+" | wc -l
+	if [ "$L" -ge "5" ]
+	then
+		echo "We assume that you know what you are doing."
+		exit 0
+	fi
+	echo "The release notes were not extended correctly (with $L lines)."
+	echo "Please make sure you added at least one line which says *(your name)* to"
```

Comment:
I would replace “says” with “states”/“mentions” here or reformulate the sentence a little bit:

 > Please make sure you add at least one line describing you contribution followed by the text “*(your name)*”.

.